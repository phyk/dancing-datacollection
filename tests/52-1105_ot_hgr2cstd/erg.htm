<!DOCTYPE html PUBLIC "-//WAPFORUM//DTD XHTML Mobile 1.2//EN" "http://www.openmobilealliance.org/tech/DTD/xhtml-mobile12.dtd">
<HTML>
<HEAD>
  <META http-equiv="Content-Type" content="text/html; charset=utf-8">
  <META name="Author" content="Hessischer Tanzsportverband e.V.">
  <META name="GENERATOR" content="TopTurnierDigital">
  <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
  <meta http-equiv="Pragma" content="no-cache" />
  <meta http-equiv="Expires" content="0" />
  <TITLE>11.05.2024 Hgr.II C Standard</TITLE>
  <link rel="stylesheet" type="text/css" href="topturnier.css">
  <meta name="viewport" content="width=device-width, initial-scale=1" />
</HEAD>
<body><div class="mainback">
<div class="eventhead">
<table border=0 width=100%>
<tr><td>11.05.2024 - OT, Hgr.II C Standard &#34;7&#34;</td><td width=30>
<a class="backbtn" href="index.htm" target="_top">&equiv;</a>
</td></tr>
</table>
</div>
<div class="maincontainer">
<div class="comphead">Ergebnis</div>
<hr class="line">
<div class="extract">
   <TABLE class="tab1">
    <TR>
     <TD class="td1" colspan="7">Endrunde</TD>
    </TR>
    <TR>
     <TD class="td2c" width="45px">Platz</TD>
     <TD class="td2">Paar/Club</TD>
     <TD class="td2cw" align=center nowrap>LW</TD>
     <TD class="td2cw" align=center nowrap>TG</TD>
     <TD class="td2cw" align=center nowrap>SF</TD>
     <TD class="td2cw" align=center nowrap>QU</TD>
     <TD class="td2" align=center nowrap>PZ</TD>
    </TR>
    <TR>
     <TD class="td3r">1.</TD>
     <TD class="td5">Kai Klede  /  Amke Beenen (519)<BR><i>TSC Erlangen d. TB 1888</i></TD>
     <TD class="td3w">51623<BR><div class="pz">2.0</div></TD>
     <TD class="td3w">32311<BR><div class="pz">1.0</div></TD>
     <TD class="td3w">61513<BR><div class="pz">1.0</div></TD>
     <TD class="td3w">51512<BR><div class="pz">1.0</div></TD>
     <TD class="td3pz">&nbsp;<br> 5.0</TD>
    </TR>
    <TR>
     <TD class="td3r">2.</TD>
     <TD class="td5">Gregor Kobsik  /  Angelina Kleiber (521)<BR><i>TSC Gr&uuml;n-Wei&szlig; Aquisgrana Aachen</i></TD>
     <TD class="td3w">42132<BR><div class="pz">1.0</div></TD>
     <TD class="td3w">51643<BR><div class="pz">3.0</div></TD>
     <TD class="td3w">43142<BR><div class="pz">2.0</div></TD>
     <TD class="td3w">42333<BR><div class="pz">3.0</div></TD>
     <TD class="td3pz">&nbsp;<br> 9.0</TD>
    </TR>
    <TR>
     <TD class="td3r">3.</TD>
     <TD class="td5">Sebastian Hauber  /  Amelie Goldfu&szlig; (516)<BR><i>TSA Schwarz-Gold d. ESV Ingolstadt</i></TD>
     <TD class="td3w">15441<BR><div class="pz">3.0</div></TD>
     <TD class="td3w">14232<BR><div class="pz">2.0</div></TD>
     <TD class="td3w">14451<BR><div class="pz">4.0</div></TD>
     <TD class="td3w">16251<BR><div class="pz">2.0</div></TD>
     <TD class="td3pz">&nbsp;<br>11.0</TD>
    </TR>
    <TR>
     <TD class="td3r">4.</TD>
     <TD class="td5">Lukas Kuschel  /  Katharina H&ouml;lzchen (523)<BR><i>TSC Schwarz-Gold im ASC G&ouml;ttingen v 1846 e.V</i></TD>
     <TD class="td3w">24364<BR><div class="pz">4.0</div></TD>
     <TD class="td3w">26465<BR><div class="pz">6.0</div></TD>
     <TD class="td3w">35235<BR><div class="pz">3.0</div></TD>
     <TD class="td3w">25444<BR><div class="pz">4.0</div></TD>
     <TD class="td3pz">&nbsp;<br>17.0</TD>
    </TR>
    <TR>
     <TD class="td3r">5.</TD>
     <TD class="td5">Dr. Felix Prihoda  /  Dr. Annemarie Prihoda (528)<BR><i>TTC Erlangen</i></TD>
     <TD class="td3w">63515<BR><div class="pz">5.0</div></TD>
     <TD class="td3w">63526<BR><div class="pz">5.0</div></TD>
     <TD class="td3w">52624<BR><div class="pz">5.0</div></TD>
     <TD class="td3w">63626<BR><div class="pz">6.0</div></TD>
     <TD class="td3pz">&nbsp;<br>21.0</TD>
    </TR>
    <TR>
     <TD class="td3r">6.</TD>
     <TD class="td5">Peter Brantsch  /  Luisa B&ouml;ck (505)<BR><i>TSC Astoria Karlsruhe</i></TD>
     <TD class="td3w">36256<BR><div class="pz">6.0</div></TD>
     <TD class="td3w">45154<BR><div class="pz">4.0</div></TD>
     <TD class="td3w">26366<BR><div class="pz">6.0</div></TD>
     <TD class="td3w">34165<BR><div class="pz">5.0</div></TD>
     <TD class="td3pz">&nbsp;<br>21.0</TD>
    </TR>
   </TABLE>
   <TABLE class="tab2" border=0>
    <TR style="height: 5px; border:0;">
     <TD class="td5" colspan=3 style="height: 5px; border:0;"></TD>
    </TR>
    <TR>
     <TD class="td1" colspan=3>2. Zwischenrunde</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">7.</TD>
     <TD class="td5">Fionn Woghen Dr. Wentorp  /  Aila Meschgbu (509)</TD>
     <TD class="td6">TSC Olsberg</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">8.</TD>
     <TD class="td5">Martin G&uuml;nther  /  Sarah P&auml;tow (514)</TD>
     <TD class="td6">TSC Astoria Karlsruhe</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">9.</TD>
     <TD class="td5">Christian Peters  /  Ronja Hormes (526)</TD>
     <TD class="td6">TSZ Blau-Gold Casino, Darmstadt</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">10.</TD>
     <TD class="td5">Nicklas Benedikt Neufang  /  Eva Eisenhardt, Laura (525)</TD>
     <TD class="td6">TSC Rot-Wei&szlig; B&ouml;blingen</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">11.</TD>
     <TD class="td5">Christoph Schl&uuml;ter  /  Franziska Gerlach (532)</TD>
     <TD class="td6">Tanzsportclub Dortmund</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">12.</TD>
     <TD class="td5">Sebastian Damm  /  Jantje Rippe (506)</TD>
     <TD class="td6">TSA d. TV Schwanewede v. 1903</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">13.</TD>
     <TD class="td5">Bernd Krauss  /  Jennifer Steuer (522)</TD>
     <TD class="td6">TSA d. TSV Schmiden</TD>
    </TR>
    <TR style="height: 5px; border:0;">
     <TD class="td5" colspan=3 style="height: 5px; border:0;"></TD>
    </TR>
    <TR>
     <TD class="td1" colspan=3>1. Zwischenrunde</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">14.- 15.</TD>
     <TD class="td5">Maximilian D&ouml;rner  /  Anita D&ouml;rner (508)</TD>
     <TD class="td6">TanzZentrum Ludwigshafen</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">14.- 15.</TD>
     <TD class="td5">Klaus Raab  /  Katalin Veszpremi (529)</TD>
     <TD class="td6">TC Blau-Orange Wiesbaden</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">16.- 17.</TD>
     <TD class="td5">Johannes Pfeiffer  /  Tamara Pfeiffer (527)</TD>
     <TD class="td6">Tanzsportclub Trier</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">16.- 17.</TD>
     <TD class="td5">Frank Thiemicke  /  Lea Offermann (534)</TD>
     <TD class="td6">TSC Astoria Karlsruhe</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">18.</TD>
     <TD class="td5">Marcel Rose  /  Imke Schwan (530)</TD>
     <TD class="td6">TC Kristall Jena</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">19.- 21.</TD>
     <TD class="td5">Carsten Beck  /  Jennifer Arnold (501)</TD>
     <TD class="td6">TC Rot-Weiss Casino Mainz</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">19.- 21.</TD>
     <TD class="td5">Markus Hajek  /  Eva Hajek (515)</TD>
     <TD class="td6">TC Rot-Weiss Casino Mainz</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">19.- 21.</TD>
     <TD class="td5">Patrick Keller  /  Stefanie Schenker (518)</TD>
     <TD class="td6">TSC dancepoint, K&ouml;nigsbrunn</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">22.- 23.</TD>
     <TD class="td5">Stefan Fischer  /  Mareike Maass (511)</TD>
     <TD class="td6">Rot-Wei&szlig;-Club Gie&szlig;en</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">22.- 23.</TD>
     <TD class="td5">Jakob Zwicker  /  Magdalena Bedner (536)</TD>
     <TD class="td6">Tanzclub Konstanz</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">24.</TD>
     <TD class="td5">Alexander Behmer  /  Dr. Juliane Scheil (502)</TD>
     <TD class="td6">Tanzsportzentrum Wetter-Ruhr</TD>
    </TR>
    <TR style="height: 5px; border:0;">
     <TD class="td5" colspan=3 style="height: 5px; border:0;"></TD>
    </TR>
    <TR>
     <TD class="td1" colspan=3>Vorrunde</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">25.</TD>
     <TD class="td5">David Schneider  /  Sonja Wendenburg (533)</TD>
     <TD class="td6">TSZ Blau-Gold Casino, Darmstadt</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">26.- 28.</TD>
     <TD class="td5">Andreas Baumeister  /  C&auml;cilia Benzin (500)</TD>
     <TD class="td6">TSC Astoria Karlsruhe</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">26.- 28.</TD>
     <TD class="td5">Achim Besler  /  Kathrin Besler (504)</TD>
     <TD class="td6">TSA d. TSG 1861 Gr&uuml;nstadt</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">26.- 28.</TD>
     <TD class="td5">Paul Wehle  /  Melanie H&ouml;schele (535)</TD>
     <TD class="td6">Tanzsportclub Balance Berlin</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">29.- 30.</TD>
     <TD class="td5">Pascal Gerbert  /  Nelly Gerbert (512)</TD>
     <TD class="td6">TSC Welfen Weingarten</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">29.- 30.</TD>
     <TD class="td5">Dirk Sch&auml;fer  /  Gertrud Lembke (531)</TD>
     <TD class="td6">TSZ Blau-Gold Casino, Darmstadt</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">31.</TD>
     <TD class="td5">Alexander Kober  /  Diana B&uuml;hren (520)</TD>
     <TD class="td6">Tanzsportclub Dortmund</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">32.</TD>
     <TD class="td5">Martin Erhardt  /  Svenja Mozian (510)</TD>
     <TD class="td6">Tanz- u. Sportzentr. Mittelrhein, Koblenz</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">33.</TD>
     <TD class="td5">Patrick Hiebl  /  Sylvia Ki&szlig;mehl (517)</TD>
     <TD class="td6">WTC Friedberg</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">34.</TD>
     <TD class="td5">Christian Deike  /  Patrycja Krohn (507)</TD>
     <TD class="td6">1. TSZ im Turn-Klubb zu Hannover</TD>
    </TR>
   </TABLE>
<br>
</div>
</div>
</div>
  <P><FONT size="1" face="Arial">Diese Liste wurde mit <A TARGET="_blank" HREF="http://www.TopTurnier.de">TopTurnier f&uuml;r Windows V9.6b</A> erstellt.<br></FONT>
</body>
</HTML>
