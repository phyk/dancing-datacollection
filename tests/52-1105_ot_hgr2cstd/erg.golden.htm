<!DOCTYPE html>
<html>
 <head>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
  <title>
   11.05.2024 Hgr.II C Standard
  </title>
 </head>
 <body>
  <div>
   <div>
    <table>
     <tr>
      <td>
       11.05.2024 - OT, Hgr.II C Standard "7"
      </td>
     </tr>
    </table>
   </div>
   <div>
    <div>
     Ergebnis
    </div>
    <hr/>
    <div>
     <table>
      <tr>
       <td>
        Endrunde
       </td>
      </tr>
      <tr>
       <td>
        Platz
       </td>
       <td>
        Paar/Club
       </td>
       <td>
        LW
       </td>
       <td>
        TG
       </td>
       <td>
        SF
       </td>
       <td>
        QU
       </td>
       <td>
        PZ
       </td>
      </tr>
      <tr>
       <td>
        1.
       </td>
       <td>
        Kai Klede / Amke Beenen (519)
        <br/>
        <i>
         TSC Erlangen d. TB 1888
        </i>
       </td>
       <td>
        51623
        <br/>
        <div>
         2.0
        </div>
       </td>
       <td>
        32311
        <br/>
        <div>
         1.0
        </div>
       </td>
       <td>
        61513
        <br/>
        <div>
         1.0
        </div>
       </td>
       <td>
        51512
        <br/>
        <div>
         1.0
        </div>
       </td>
       <td>
        <br/>
        5.0
       </td>
      </tr>
      <tr>
       <td>
        2.
       </td>
       <td>
        Gregor Kobsik / Angelina Kleiber (521)
        <br/>
        <i>
         TSC Grün-Weiß Aquisgrana Aachen
        </i>
       </td>
       <td>
        42132
        <br/>
        <div>
         1.0
        </div>
       </td>
       <td>
        51643
        <br/>
        <div>
         3.0
        </div>
       </td>
       <td>
        43142
        <br/>
        <div>
         2.0
        </div>
       </td>
       <td>
        42333
        <br/>
        <div>
         3.0
        </div>
       </td>
       <td>
        <br/>
        9.0
       </td>
      </tr>
      <tr>
       <td>
        3.
       </td>
       <td>
        Sebastian Hauber / Amelie Goldfuß (516)
        <br/>
        <i>
         TSA Schwarz-Gold d. ESV Ingolstadt
        </i>
       </td>
       <td>
        15441
        <br/>
        <div>
         3.0
        </div>
       </td>
       <td>
        14232
        <br/>
        <div>
         2.0
        </div>
       </td>
       <td>
        14451
        <br/>
        <div>
         4.0
        </div>
       </td>
       <td>
        16251
        <br/>
        <div>
         2.0
        </div>
       </td>
       <td>
        <br/>
        11.0
       </td>
      </tr>
      <tr>
       <td>
        4.
       </td>
       <td>
        Lukas Kuschel / Katharina Hölzchen (523)
        <br/>
        <i>
         TSC Schwarz-Gold im ASC Göttingen v 1846 e.V
        </i>
       </td>
       <td>
        24364
        <br/>
        <div>
         4.0
        </div>
       </td>
       <td>
        26465
        <br/>
        <div>
         6.0
        </div>
       </td>
       <td>
        35235
        <br/>
        <div>
         3.0
        </div>
       </td>
       <td>
        25444
        <br/>
        <div>
         4.0
        </div>
       </td>
       <td>
        <br/>
        17.0
       </td>
      </tr>
      <tr>
       <td>
        5.
       </td>
       <td>
        Dr. Felix Prihoda / Dr. Annemarie Prihoda (528)
        <br/>
        <i>
         TTC Erlangen
        </i>
       </td>
       <td>
        63515
        <br/>
        <div>
         5.0
        </div>
       </td>
       <td>
        63526
        <br/>
        <div>
         5.0
        </div>
       </td>
       <td>
        52624
        <br/>
        <div>
         5.0
        </div>
       </td>
       <td>
        63626
        <br/>
        <div>
         6.0
        </div>
       </td>
       <td>
        <br/>
        21.0
       </td>
      </tr>
      <tr>
       <td>
        6.
       </td>
       <td>
        Peter Brantsch / Luisa Böck (505)
        <br/>
        <i>
         TSC Astoria Karlsruhe
        </i>
       </td>
       <td>
        36256
        <br/>
        <div>
         6.0
        </div>
       </td>
       <td>
        45154
        <br/>
        <div>
         4.0
        </div>
       </td>
       <td>
        26366
        <br/>
        <div>
         6.0
        </div>
       </td>
       <td>
        34165
        <br/>
        <div>
         5.0
        </div>
       </td>
       <td>
        <br/>
        21.0
       </td>
      </tr>
     </table>
     <table>
      <tr>
       <td>
        2. Zwischenrunde
       </td>
      </tr>
      <tr>
       <td>
        7.
       </td>
       <td>
        Fionn Woghen Dr. Wentorp / Aila Meschgbu (509)
       </td>
       <td>
        TSC Olsberg
       </td>
      </tr>
      <tr>
       <td>
        8.
       </td>
       <td>
        Martin Günther / Sarah Pätow (514)
       </td>
       <td>
        TSC Astoria Karlsruhe
       </td>
      </tr>
      <tr>
       <td>
        9.
       </td>
       <td>
        Christian Peters / Ronja Hormes (526)
       </td>
       <td>
        TSZ Blau-Gold Casino, Darmstadt
       </td>
      </tr>
      <tr>
       <td>
        10.
       </td>
       <td>
        Nicklas Benedikt Neufang / Eva Eisenhardt, Laura (525)
       </td>
       <td>
        TSC Rot-Weiß Böblingen
       </td>
      </tr>
      <tr>
       <td>
        11.
       </td>
       <td>
        Christoph Schlüter / Franziska Gerlach (532)
       </td>
       <td>
        Tanzsportclub Dortmund
       </td>
      </tr>
      <tr>
       <td>
        12.
       </td>
       <td>
        Sebastian Damm / Jantje Rippe (506)
       </td>
       <td>
        TSA d. TV Schwanewede v. 1903
       </td>
      </tr>
      <tr>
       <td>
        13.
       </td>
       <td>
        Bernd Krauss / Jennifer Steuer (522)
       </td>
       <td>
        TSA d. TSV Schmiden
       </td>
      </tr>
      <tr>
       <td>
        1. Zwischenrunde
       </td>
      </tr>
      <tr>
       <td>
        14.- 15.
       </td>
       <td>
        Maximilian Dörner / Anita Dörner (508)
       </td>
       <td>
        TanzZentrum Ludwigshafen
       </td>
      </tr>
      <tr>
       <td>
        14.- 15.
       </td>
       <td>
        Klaus Raab / Katalin Veszpremi (529)
       </td>
       <td>
        TC Blau-Orange Wiesbaden
       </td>
      </tr>
      <tr>
       <td>
        16.- 17.
       </td>
       <td>
        Johannes Pfeiffer / Tamara Pfeiffer (527)
       </td>
       <td>
        Tanzsportclub Trier
       </td>
      </tr>
      <tr>
       <td>
        16.- 17.
       </td>
       <td>
        Frank Thiemicke / Lea Offermann (534)
       </td>
       <td>
        TSC Astoria Karlsruhe
       </td>
      </tr>
      <tr>
       <td>
        18.
       </td>
       <td>
        Marcel Rose / Imke Schwan (530)
       </td>
       <td>
        TC Kristall Jena
       </td>
      </tr>
      <tr>
       <td>
        19.- 21.
       </td>
       <td>
        Carsten Beck / Jennifer Arnold (501)
       </td>
       <td>
        TC Rot-Weiss Casino Mainz
       </td>
      </tr>
      <tr>
       <td>
        19.- 21.
       </td>
       <td>
        Markus Hajek / Eva Hajek (515)
       </td>
       <td>
        TC Rot-Weiss Casino Mainz
       </td>
      </tr>
      <tr>
       <td>
        19.- 21.
       </td>
       <td>
        Patrick Keller / Stefanie Schenker (518)
       </td>
       <td>
        TSC dancepoint, Königsbrunn
       </td>
      </tr>
      <tr>
       <td>
        22.- 23.
       </td>
       <td>
        Stefan Fischer / Mareike Maass (511)
       </td>
       <td>
        Rot-Weiß-Club Gießen
       </td>
      </tr>
      <tr>
       <td>
        22.- 23.
       </td>
       <td>
        Jakob Zwicker / Magdalena Bedner (536)
       </td>
       <td>
        Tanzclub Konstanz
       </td>
      </tr>
      <tr>
       <td>
        24.
       </td>
       <td>
        Alexander Behmer / Dr. Juliane Scheil (502)
       </td>
       <td>
        Tanzsportzentrum Wetter-Ruhr
       </td>
      </tr>
      <tr>
       <td>
        Vorrunde
       </td>
      </tr>
      <tr>
       <td>
        25.
       </td>
       <td>
        David Schneider / Sonja Wendenburg (533)
       </td>
       <td>
        TSZ Blau-Gold Casino, Darmstadt
       </td>
      </tr>
      <tr>
       <td>
        26.- 28.
       </td>
       <td>
        Andreas Baumeister / Cäcilia Benzin (500)
       </td>
       <td>
        TSC Astoria Karlsruhe
       </td>
      </tr>
      <tr>
       <td>
        26.- 28.
       </td>
       <td>
        Achim Besler / Kathrin Besler (504)
       </td>
       <td>
        TSA d. TSG 1861 Grünstadt
       </td>
      </tr>
      <tr>
       <td>
        26.- 28.
       </td>
       <td>
        Paul Wehle / Melanie Höschele (535)
       </td>
       <td>
        Tanzsportclub Balance Berlin
       </td>
      </tr>
      <tr>
       <td>
        29.- 30.
       </td>
       <td>
        Pascal Gerbert / Nelly Gerbert (512)
       </td>
       <td>
        TSC Welfen Weingarten
       </td>
      </tr>
      <tr>
       <td>
        29.- 30.
       </td>
       <td>
        Dirk Schäfer / Gertrud Lembke (531)
       </td>
       <td>
        TSZ Blau-Gold Casino, Darmstadt
       </td>
      </tr>
      <tr>
       <td>
        31.
       </td>
       <td>
        Alexander Kober / Diana Bühren (520)
       </td>
       <td>
        Tanzsportclub Dortmund
       </td>
      </tr>
      <tr>
       <td>
        32.
       </td>
       <td>
        Martin Erhardt / Svenja Mozian (510)
       </td>
       <td>
        Tanz- u. Sportzentr. Mittelrhein, Koblenz
       </td>
      </tr>
      <tr>
       <td>
        33.
       </td>
       <td>
        Patrick Hiebl / Sylvia Kißmehl (517)
       </td>
       <td>
        WTC Friedberg
       </td>
      </tr>
      <tr>
       <td>
        34.
       </td>
       <td>
        Christian Deike / Patrycja Krohn (507)
       </td>
       <td>
        1. TSZ im Turn-Klubb zu Hannover
       </td>
      </tr>
     </table>
     <br/>
    </div>
   </div>
  </div>
 </body>
</html>
